tests/browser/karma.js

```diff
@@ -0,0 +1,23 @@
+const { output: { file } } = require('./rollup.js')
+
+module.exports = (config) => {
+  const { LOG_INFO: logLevel } = config
+
+  config.set({
+    browsers: ['Chrome', 'Firefox'],
+    frameworks: ['mocha'],
+    port: 9876,
+    logLevel,
+    singleRun: true,
+    concurrency: 1,
+    hooks: [
+      'karma-chrome-launcher',
+      'karma-firefox-launcher',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/omrilotan/isbot/pullRequest?prid=7728710)
```suggestion
      "karma-firefox-launcher",
```